db/migrate/20170110132223_add_unlimited_participants_to_events.rb

```diff
@@ -0,0 +1,5 @@
+class AddUnlimitedParticipantsToEvents < ActiveRecord::Migration
+  def change
+    add_column :events, :participants_are_unlimited, :boolean, default: false
```

Comment:
@bjrne The default value was added recently during the PR, so I guess the migration wasnt run again after that change, and no one of us kept in mind that the `schema.rb` needs to be updated too.
TL;DR default value should be there, we will fix this soon